core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/data/impl/DataFeedServiceImpl.java

```diff
@@ -294,7 +294,7 @@ private void printViewItemToCsv(CSVPrinter csvPrinter, GridViewItem viewItem) {
                     CsvUtil.buildLocation(viewItem.getResourcePath(), viewItem.getPropertyName())
             );
         } catch (IOException e) {
-            LOG.error(String.format("Failed to build CSV for the grid resource %s", viewItem.getValue()), e);
+            LOG.info(String.format("Failed to build CSV for the grid resource %s", viewItem.getValue()), e);
```

Comment:
Same here, + please use the standard `{}` format for placeholders

Reply:
done